src/core/stream/live_events_handler.ts

```diff
@@ -29,7 +29,7 @@ import EVENTS, {
  * @param {Object} manifest
  * @returns {Observable}
  */
-function refreshManifest(
+export function refreshManifest(
```

Comment:
If ``refreshManifest`` is exported, it should not be defined in a file named ``live_events_handler.ts`` anymore